.github/workflows/CI.yml

```diff
@@ -86,4 +86,4 @@ jobs:
       - name: Run formatting checks
```

Comment:
Do we want to run these checks/linters on `testing_containers` too? I just realized that we don't check `ever_given` either

Reply:
Yes ever_given but not testing_containers imho. But actually we should build the testing_containers automatically, and that would be the best place to run black too